File: BOJ/4344.py
import sys
sys.stdin = open('input.txt',"rt")

n = int(sys.stdin.readline().rstrip())


for i in range(n):
    input_data = list(map(int,sys.stdin.readline().rstrip().split()))
    avg = sum(input_data[1:]) / input_data[0]
    cnt = 0
    for j in range(1,input_data[0]+1):
        if input_data[j] > avg:
            cnt += 1
    else:
        rate = round(100*cnt/input_data[0],3)
        print(f"{rate:.3f}%")

File: BOJ/4344.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*
fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val n = br.readLine().toInt()

    for (i in 1..n){
        var list = br.readLine().split(' ').map{it.toInt()}
        val avg = (list.sum() - list[0])/list[0]
        var cnt = 0
        for(i in 1..list[0]){
            if (list[i] > avg){
                cnt += 1
            }
        }
        print(String.format("%.3f",(cnt*100/list[0].toDouble())))
        println("%")

        //var a = "%.3f".format((cnt/list[0])*100)
        //print("${a}%\n")
        //print(", ")
        //print("%.3f".format((cnt/list[0])*100)+"%\n")
    }


}